### ISSUE TITLE
Unwanted debugging message with nerdctl

### ISSUE THREAD

**Issue by gunamata** [rancher-sandbox/rancher-desktop#639]
After @jandubois 's #631 , I tried the nerctl command `nerdctl.exe run -p 5000:5000 -it simple-flask-app:v1.0` again. 
Earlier argument parsing message is gone. Now, I see below message in the console.

`No command "container run simple-flask-app:v1.0", ignoring positional arguments`

After a quick chat with @mook-as , I understand this is an unwanted debugging message that should have been silenced by default.

**Comment by jandubois** [comment]
I would have just deleted it, or how would you want to "silence" it @mook-as ?

There is another logging call that looks like a debugging tool too: `log.Printf("Using custom parse handler")`. Should that also be removed?

**Comment by mook-as** [comment]
Yeah, just removing it is fine. Trying to do debug logging isn't worth it here, probably.
